README.md

```diff
@@ -6,15 +6,44 @@
 
 HighTeach is a social network which enables tutors to expose themselves through dynamic connections with potential students, based on fields or special needs.
 
+## Getting Started
+
+### Prerequisites
+
+Install [Vagrant](https://www.vagrantup.com/).
+
+> Vagrant is a tool for using virtual machines to share development environments.
+
+Install [Virtual Box](https://www.virtualbox.org/).
+
+> VirtualBox is a hypervisor, which lets us create and run virtual machines.
+
+### Initializing virtual environment
+
+- Clone the HighTeach repository Using the command `git clone https://github.com/redhat-beyond/HighTeach.git`
+- Open command prompt in HighTeach directory
+- Spin the environment using `vagrant up`
+- Go to http://localhost:8000 in your web browser
+
+### Additional important Virtual environment commands:
+
+- You can enter the virtual machine by running `vagrant ssh`
+- Use `Vagrant halt` to shut the environment down
+- When finished, run `vagrant destroy -f` to tear down the environment
```

Comment:
Why do you need to use `-f` flag here?

Reply:
Hi, The -f flag is not required, will remove it from the code block

Reply:
done